packages/frontend/serverless.yml

```diff
@@ -15,7 +15,7 @@ provider:
   region: ${opt:region}
 
 custom:
-  app: ${file(../app.yml)}
+  app: ${file(../../app.yml)}
```

Comment:
Does this work, even though `projectDir` is set to `..`? 
I thought that Serverless Framework mandated now that files could only be loaded from within `projectDir`.
I'm guessing it does work, but maybe with a warning?

Reply:
That feels like a change in behaviour, maybe since sls v3. The newer resolved in later 2.x versions did not allow access outside projectDir. Apparently, projectDir will be deprecated in v4 (https://www.serverless.com/framework/docs/deprecations) so we can start to remove it, it seems.